src/partials/advantages.html

```diff
@@ -1,9 +1,29 @@
-<section class="advantages">
-  <h2>advantages</h2>
-  <h3>721</h3>
-  <p>Aliquam ac dui vel dui vulputate consectetur. Mauris massa.</p>
-  <h3>16kg</h3>
-  <p>Aliquam ac dui vel dui vulputate consectetur. Mauris massa.</p>
-  <h3>84</h3>
-  <p>Aliquam ac dui vel dui vulputate consectetur. Mauris massa.</p>
+<section class="advantages" id="advantages">
+  <h2 class="advantages__header-hidden">advantages</h2>
+  <ul class="advantages__list">
+    <li class="advantages__list-header advantages__list-header--1">
+      721
+      <ul>
```

Comment:
ul robisz dla listy, która ma co najmniej 2 li aby była listą, a tu masz tylo jeden li więc wystarczy że to usuniesz i zostawisz samo p

Reply:
Hej, muszę użyć listy zagnieżdżonej bo to jest lista w liście i bez tego wszystko się rozjeżdża - patrz załączone screeny. ikona + header + text to jest jedna lista i tych list są trzy i są one zagnieżdżone w nadrzędnej liście advantages__list która zawiera te trzy listy. W konspekcie było o listach zagnieżdżonych. Szukałam na internecie informacji czy lista musi zawierać conajmniej dwa elementy li, albo że nie może zawierać tylko jednego, ale nic nie znalazłam na ten temat, więc nie ma powodu aby to zmieniać.

![image](https://user-images.githubusercontent.com/117306131/215150331-fb4bce35-14a5-4a9c-b9e4-d21a85f6445b.png)

![image](https://user-images.githubusercontent.com/117306131/215151119-10297290-1c67-4d07-b134-69d879b801c1.png)